ykrt/src/mt.rs

```diff
@@ -172,10 +179,11 @@ impl MT {
 
     /// Queue `job` to be run on a worker thread.
     fn queue_job(self: &Arc<Self>, job: Box<dyn FnOnce() + Send>) {
+        #[cfg(yk_llvm_sync_hack)]
+        self.active_worker_jobs.fetch_add(1, Ordering::Relaxed);
         // We have a very simple model of worker threads. Each time a job is queued, we spin up a
         // new worker thread iff we aren't already running the maximum number of worker threads.
         // Once started, a worker thread never dies, waiting endlessly for work.
-
```

Comment:
That should have been a separate commit.

I won't make you change it :smile:

Reply:
In retrospect, moving the comment would have been better. Fixed in 3598efc.